lib/tasks/bus_stops_route_creation.rake

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops_route do
+  task :import => :environment do
+    stop_hash = {}
+    missing_trips = []
+
+    CSV.foreach('trips.txt', headers: true) do |row|
+      route_number = row['route_id']
+      direction = row['direction_id']
+      trip_id = row['trip_id']
+      unless route_number.blank? 
```

Comment:
Perhaps a comment explaining why this might be (I know, but future others may not)

Reply:
I don't know if I can explain it. There are visually no blank rows in this CSV, but I'm on a blank row when `route_number` is blank (everything else is just `nil`, but `route_number1` is a blank string.

Reply:
It only happens once btw.

Reply:
What if it's a hidden character type deal, like when the zero-width space character was causing me all sorts of headaches?

Reply:
Ah ok